src/hipscat/io/file_io/file_pointers.py

```diff
@@ -0,0 +1,35 @@
+import os
+from typing import NewType
+
+FilePointer = NewType("FilePointer", str)
+
+
+def get_file_pointer_from_path(path: str) -> FilePointer:
+    """Returns a file pointer from a path string"""
+    return FilePointer(path)
+
+
+def append_paths_to_pointer(pointer: FilePointer, *paths: str) -> FilePointer:
+    """Append directory or file names to a specified file pointer.
```

Comment:
Should this be "Append directories and a file name"? It seems like there could be multiple directories, but only one filename.

Reply:
Yes, you're right, I'll update that